apps/product/admin.py

```
@@ -1,7 +1,144 @@
+@admin.register(PriorityShipments)
+class PriorityShipmentsrAdminView(admin.ModelAdmin):
+    
+    list_display =["id", "vendor_code", "travel_days","arrive_days", "marketplace_type", "sales", "sales_share","shipments_share","shipping_priority"]
+    search_fields = ["warehouse__id"]
+    list_filter = ["marketplace_type"]
+
+    def vendor_code(self, recommandations: PriorityShipments):
+        return recommandations.warehouse.region_name or recommandations.warehouse.oblast_okrug_name
+
+@admin.register(Shipment)
+class PriorityShipmentsrAdminView(admin.ModelAdmin):
```

Ruff

Redefinition of unused `PriorityShipmentsrAdminView` from line 116

(F811)